api/test/utils.js

```diff
@@ -77,3 +96,81 @@ export function getSigningOptions(ctx) {
     bucket: ctx.bucketName,
   }
 }
+
+/** @returns {Promise<MockAccess>} */
+export async function createAccessServer () {
```

Comment:
Might be nice to consider creating a mock module for this as I expect it to be needed in more services. Not for now, just raising thoughts for visibility :)